libobs/obs-nix-drm.c

```diff
@@ -0,0 +1,79 @@
+/******************************************************************************
+    Copyright (C) 2022 by Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include "obs-internal.h"
+#include "obs-nix-platform.h"
+#include "obs-nix-drm.h"
+
+#include <unistd.h>
+#include <sys/mman.h>
+#include <string.h>
+
+struct obs_hotkeys_platform {
+};
+
+void obs_nix_drm_log_info(void) {}
+
+static bool obs_nix_drm_hotkeys_platform_init(struct obs_core_hotkeys *hotkeys)
```

Comment:
What is the plan for supporting hotkeys?

Reply:
Global shortcuts portal when running under a compositor, libinput otherwise. This is a very far fetched idea though, I haven't spend any time thinking about this. The use case for this renderer is to export the resulting frames as a shared texture, potentially for offscreen rendering or integrating with other exotic cases. The texture sharing part is missing from this PR - I have some working code in a branch, but it's not ready to be published yet.